### DISCUSSION TITLE
Possible to fine with an other language

### DISCUSSION THREAD

**Discussion by ahoepf** [metavoiceio/metavoice-src#123] [Q&A]
Hi all, 

do someone know, if it is possible to finetune with an other language like german?

greeting

Al

**Comment by vatsalaggarwal** [comment]
Yes, the Kotoba (https://github.com/kotoba-tech/kotoba-speech-release) team finetuned the model to make it work in Japanese. My understanding is that they used the pretrained first stage model as a base model, and only finetuned this. The rest was kept fixed.
